Profile/__test__/handler.test.ts

```diff
@@ -27,6 +29,12 @@ const aValidUser: HslJwtPayloadExtended = {
   name: "name" as NonEmptyString,
   spid_level: SpidLevel.L2
 };
+const aValidExchangeUser: ExchangeJwtPayloadExtended = {
```

Comment:
may I suggest to put this in `config.mock.ts` and importing it among all tests that needs it? in this PR i saw this exact object declared in 3 different test files